resources/lib/library.py

```diff
@@ -1494,8 +1494,8 @@ def explorer( self, history, location, label, thumbnail, itemType, isWidget = Fa
                     listitem.setProperty( "widgetPath", location )
 
                 elif itemType == "32011" or itemType == "32019" or itemType == "32073":
-                    action = 'ActivateWindow(10501,"' + location + '",return)'
-                    listitem.setProperty( "windowID", "10501" )
+                    action = 'ActivateWindow(10502,"' + location + '",return)'
+                    listitem.setProperty( "windowID", "10502" )
```

Comment:
in general there should be no need to mess with the window ids at all. The preferred way for calling / referencing windows is by using their alias ("musiclibrary" in this case)